tests/functional/sync_force_auth.js

```diff
@@ -46,22 +47,16 @@ define([
       return this.remote
         .then(setupTest(this, true))
 
-        // add a slight delay to ensure the page does not transition
-        .sleep(1000)
-
-        // the page does not transition.
-        .then(testElementExists('#fxa-force-auth-header'))
-        .then(testIsBrowserNotified(this, 'can_link_account'))
-        .then(testIsBrowserNotified(this, 'login'));
+        // page does not transition
+        .sleep(2000)
```

Comment:
I notice `sleep(2000)` is hard-coded in quite a few places. Is there a case for extracting it to a common, appropriately-named function so that we update all of the related sleeps in one go in the future?